.gitignore

```diff
@@ -0,0 +1,39 @@
+# intellij ide config folders
+.idea/
+
+# node dependence folder
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+node_modules/
+
+# environment variable file
+.env
+
+# lock dependence
+package-lock.json
+yarn.json
```

Comment:
Great work @llwasampijja.
I think the yarn.json file should be replaced by yarn.lock

Reply:
@JamesMudidi  thanks for the review, I think I will just include the yarn.lock and leave the yarn.json to carter for individuals who may choose to use yarn to manage dependencies

Reply:
I think that will be great.